src/main/java/com/epam/lab/payments/model/ApiController.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.lab.payments.model;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class ApiController {
+
+    private final CardRepository cardRepository;
```

Comment:
It's a bad practice - inject repository in controller. 
Right way 
Controller -> Service -> Repository

Service - wrapper around repository or Dao layer, u should google it. For example - https://softwareengineering.stackexchange.com/questions/220909/service-layer-vs-dao-why-both

Reply:
This is also temporal staff, just for quickness. Will be replaced later with a real REST service